prime-router/metadata/fhir_mapping/hl7/datatypes/CX/Identifier.yml

```diff
@@ -0,0 +1,177 @@
+resourceType: Identifier
+
+value:
+  type: STRING
+  valueOf: CX.1
+  expressionType: HL7Spec
+
+extension_1:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: identifier-check-digit
+    valueString:
+      type: STRING
+      expressionType: HL7Spec
+      valueOf: CX.2
+
+extension_2:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: naming-system-check-digit
+    valueString:
+      type: STRING
+      expressionType: HL7Spec
+      valueOf: CX.3
+
+# Checking if HD is an Organization or a System by looking at the ID type.
+# If ID type is null or ID type is not ISO or UUID, treat as organization.
+# Otherwise, treat as system.
+assigner_1:
+  vars:
+    cx43: String, CX.4.3
+  condition: cx43 NULL
+  valueOf: datatypes/HD/Organization
+  expressionType: reference
+  specs: CX.4
+
+assigner_2:
+  vars:
+    cx43: String, CX.4.3
+  condition: $cx43 NOT_NULL && $cx43 NOT_EQUALS ISO && $cx43 NOT_EQUALS UUID
+  valueOf: datatypes/HD/Organization
+  expressionType: reference
+  specs: CX.4
+
+system:
+  vars:
+    cx41: String, CX.4.1
+    cx43: String, CX.4.3
+  expressionType: nested
+  condition: $cx43 EQUALS ISO || cx43 EQUALS UUID
+  expressions:
+    - condition: $cx41 NOT_NULL
+      expressionType: HL7Spec
+      type: STRING
+      valueOf: CX.4.1
+    - condition: $cx41 NULL && cx43 EQUALS ISO
+      expressionType: HL7Spec
+      type: STRING
+      constants:
+        prefix: "urn:oid:"
+      vars:
+        uri: String, CX.4.2
+        join: $prefix + $uri
+      valueOf: $join
+    - condition: $cx41 NULL && cx43 EQUALS UUID
+      expressionType: HL7Spec
+      type: STRING
+      constants:
+        prefix: "urn:uuid:"
+      vars:
+        uri: String, CX.4.2
+        join: $prefix + $uri
+      valueOf: $join
+
+extension_system:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: coding-system
+    valueCodeableConcept:
+      valueOf: datatype/CodeableConcept_var
+      expressionType: resource
+      vars:
+        code: String, CX.4
+
+type:
+  valueOf: datatype/CodeableConcept_var
+  expressionType: resource
+  vars:
+    code: $code
+    system: $system
+    text: $text
```

Comment:
Don't think we need `text` here. Also it might be good to leave a comment with this element to explain why it's here, and maybe even put it at the end after all of the CX fields. Optional, but it took me a minute to figure it out, so it might help.